### DISCUSSION TITLE
using bytes as input to AudioFileClip instead of audio path

### DISCUSSION THREAD

**Discussion by fartoot** [Zulko/moviepy#2054] [Q&A]
Hey,

The function generate_audio() takes text as input and returns bytes b'\xff\xfb\x90\xc4\x00\x00\x10\x85\x96\xff\...'
I want to use this bytes audio directly rather than save it on disk.

```
bytes_output = tools.generate_audio("hello, man im james") # returns b'\xff\xfb\x90\xc4\x00\x00\x10\x85\x96\xff\...'
audio_clip = AudioFileClip(bytes_output)
```

btw, I have tried BytesIO but it is not working

**Comment by Glenn-Chiang** [comment]
facing the same issue but as usual, can't find any documentation or discussion on this
